scripts/pre_push_hook.py

```diff
@@ -59,6 +59,12 @@
 
 
 class ChangedBranch(object):
+     """class for changing branch in which modified file is present.
+    
```

Comment:
Change to -> Class for changing the branch in which modified file is present.

Reply:
relsolved it